src/util/manifest.js

```diff
@@ -83,3 +84,11 @@ export function getManifestId(manifestData: ExtensionManifest): string | void {
   return manifestData.applications ?
     manifestData.applications.gecko.id : undefined;
 }
+
+function stripBOM(content) {
```

Comment:
I think we need a more general approach to stripping the BOM since it applies to more than just manifest files. I suggest searching for this:

```js
import parseJSON from 'parse-json'
```

Next, create a `web-ext` wrapper for it called `parseJSON` (the same name) and put it somewhere like `src/util/json.js`. Make that function strip the BOM from any JSON file. Replace all old calls to `parseJSON` with the new one.